skiko/src/wasmMain/kotlin/org/jetbrains/skia/impl/Managed.wasm.kt

```diff
@@ -0,0 +1,18 @@
+package org.jetbrains.skia.impl
+
+internal external class FinalizationRegistry(cleanup: (FinalizationThunk) -> Unit) {
```

Comment:
It relies on js https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/FinalizationRegistry, right?

Reply:
Yes, GC for wasm objects works exactly in same way as for JS, so we can use it for wasm as well.